app/assets/javascripts/discourse/app/components/search-menu/results/recent-searches.gjs

```diff
@@ -53,11 +53,11 @@ export default class RecentSearches extends Component {
       <div class="search-menu-recent">
         <div class="heading">
           <h4>{{i18n "search.recent"}}</h4>
-          <FlatButton
+          <DButton
             @title="search.clear_recent"
             @icon="xmark"
             @action={{this.clearRecent}}
-            class="clear-recent-searches"
+            class="btn-flat no-text clear-recent-searches"
```

Comment:
`no-text` class can be removed here, this class is applied within `DButton` logic